test/extensions/filters/listener/common/fuzz/listener_filter_fakes.h

```diff
@@ -0,0 +1,95 @@
+#include "common/api/os_sys_calls_impl.h"
+#include "common/network/io_socket_handle_impl.h"
+#include "common/network/utility.h"
+
+#include "test/mocks/network/mocks.h"
+
+#include "gmock/gmock.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ListenerFilters {
+
+class FakeConnectionSocket : public Network::MockConnectionSocket {
+public:
+  FakeConnectionSocket()
+      : io_handle_(std::make_unique<Network::IoSocketHandleImpl>(42)), local_address_(nullptr),
```

Comment:
Would it be better to set 42 as a const? I notice there are some other places in this commit where 42 is also used.

Reply:
Yup, great idea! That should make it much easier to change the fake socket's fd in the future, if needed.

Reply:
Haha, 42. Prefer maybe kConstantVar as a static constexpr over a macro? Just in case.